sql/mob_groups.sql

```diff
@@ -14665,16 +14665,18 @@ INSERT INTO `mob_groups` VALUES (26,0,271,'Catacomb_Marolith',0,128,0,0,0,0,0,0)
 INSERT INTO `mob_groups` VALUES (27,0,271,'Sepulchral_Lizard',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (28,0,271,'Wartless_Toad',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (29,0,271,'Ulcerous_Acuex',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (30,0,271,'Unfettered_Twitherym',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (31,0,271,'Supernal_Chapuli',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (32,0,271,'Transcendent_Scorpion',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (33,0,271,'Mastop',0,128,0,0,0,0,0,0);
-INSERT INTO `mob_groups` VALUES (34,0,271,'Taxet',0,128,0,0,0,0,0,0);
+
+INSERT INTO `mob_groups` VALUES (30,4862,271,'Unfettered_Twitherym',0,128,0,80000,9999,110,110,0);
+INSERT INTO `mob_groups` VALUES (31,4863,271,'Supernal_Chapuli',0,128,0,83000,9999,110,110,0);
+INSERT INTO `mob_groups` VALUES (32,6911,271,'Transcendent_Scorpion',0,128,0,96500,9999,110,110,0);
+INSERT INTO `mob_groups` VALUES (33,6912,271,'Mastop',0,128,0,70500,9999,110,110,0);
+INSERT INTO `mob_groups` VALUES (34,4864,271,'Taxet',0,128,0,100000,9999,119,119,0);
 INSERT INTO `mob_groups` VALUES (35,0,271,'Muyingwa',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (36,0,271,'Fugacious_Eruca',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (37,0,271,'Fugacious_Beetle',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (38,0,271,'Fugacious_Diremite',0,128,0,0,0,0,0,0);
 INSERT INTO `mob_groups` VALUES (39,0,271,'Fugacious_Luckybug',0,128,0,0,0,0,0,0);
+
```

Comment:
What are these gaps for?

Reply:
i had fixed it. i think auto idented em by mistake